internal/blockdb/migrate.go

```diff
@@ -80,7 +84,21 @@ ON CONFLICT(git_sha) DO UPDATE SET git_sha=git_sha`, nowRFC3339(), gitSha)
 		return fmt.Errorf("create table tx: %w", err)
 	}
 
-	_, err = db.Exec(`DROP VIEW IF EXISTS v_tx_flattened`)
+	_, err = db.Exec(`ALTER TABLE chain ADD COLUMN chain_type TEXT NOT NULL check(length(chain_type) > 0) DEFAULT "cosmos"`)
+	if errIgnoreDuplicateColumn(err, "chain_type") != nil {
+		return fmt.Errorf("alter table chain add chain_type: %w", err)
+	}
+
+	// Creating views should be last migration step.
+	// Error already wrapped.
+	return upsertViews(db)
```

Comment:
`Migrate(...)` was getting long, so abstracted views into another function. May help ensure views are last in the migration process.